pandas/core/tools/datetimes.py

```diff
@@ -434,9 +449,11 @@ def _convert_listlike(arg, box, format, name=None, tz=tz):
                     yearfirst=yearfirst,
                     require_iso8601=require_iso8601
                 )
-
             if is_datetime64_dtype(result) and box:
                 result = DatetimeIndex(result, tz=tz, name=name)
+            # GH 6415
+            elif arg_is_series and utc:
```

Comment:
So my suggestion is: can't we just also for Series return a DatetimeIndex (which already handles the utc keyword fine), and then afterwards just convert this to a Series as we now convert the array to a Series. 
(but it might be that there are other reasons for having it as an array that might break, I don't know)